workers/unity/Packages/io.improbable.gdk.buildsystem/Configuration/BuildConfigEditor.cs

```diff
@@ -632,28 +633,56 @@ private GUIContent GetBuildTargetGuiContents(BuildTargetConfig c)
 
         private void DrawBuildTargets(BuildEnvironmentConfig env, int hash)
         {
+            // Build a mapping the build target position in the toolbar and it's position in the BuildTargets array.
+            // Sometimes there is a different of targets in the BuildTargets array and those in the toolbar as
+            // some are deprecated
+            if (toolbarMapping.Count == 0)
+            {
+                var index = 0;
+                var skipped = 0;
+                env.BuildTargets.ForEach(t =>
```

Comment:
I think you can make this a normal for loop to make it a little clearer. `skipped` will become the for index.